docs/openapi-ts/configuration.md

```diff
@@ -474,6 +474,101 @@ export default {
 };
 ```
 
+## Fix Schema
+
+If you need to modify schemas in your OpenAPI specification before code generation, you can use `input.fix.schema` to apply custom transformations to specific schemas.
+
+You can provide fix functions for individual schemas by their names. Each function receives the schema object and can modify it directly.
+
+::: code-group
+
+```js [date-time to timestamp]
+export default {
+  input: {
+    fix: {
```

Comment:
What do you think about calling this `patch`?

```suggestion
    patch: {
```

Reply:
That sounds good to me. I'll make the changes.